From e32008e26cde6df38e46c2827f2a26df7bc13d60 Mon Sep 17 00:00:00 2001
From: Dominik Poggel <pog@iesy.com>
Date: Mon, 5 Jun 2023 09:17:04 +0200
Subject: [PATCH] arm64: dts: iesy: change i2c eeprom addr

The eeprom changes it's address 51-->53 in favor of the new added RTC on Rev2 of the base board
---
 arch/arm64/boot/dts/iesy/iesy-imx8mm-eva-mi.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/iesy/iesy-imx8mm-eva-mi.dts b/arch/arm64/boot/dts/iesy/iesy-imx8mm-eva-mi.dts
index 2aa4df477d52..ee64804b8d3c 100644
--- a/arch/arm64/boot/dts/iesy/iesy-imx8mm-eva-mi.dts
+++ b/arch/arm64/boot/dts/iesy/iesy-imx8mm-eva-mi.dts
@@ -164,9 +164,9 @@ sensor@4e {
         vs-supply = <&reg_3v3_carrier>;
 	};
 
-	eeprom@51 {
+	eeprom@53 {
 		compatible = "atmel,24c64";
-		reg = <0x51>;
+		reg = <0x53>;
 		pagesize = <64>;
         vcc-supply = <&reg_1v8_carrier>;
 	};
